.github/workflows/ci_tests_dev.yaml

```
@@ -118,6 +118,10 @@ jobs:
           python setup.py sdist --formats=zip
           pip install dist/*
 
+      - name: Add GMT's bin to PATH (Linux/macOS)
+        run: echo ${GITHUB_WORKSPACE}/gmt-install-dir/bin >> $GITHUB_PATH
+        if: runner.os != 'Windows'
+
       # Run the tests
       - name: Test with pytest (Linux/macOS)
         run: make test PYTEST_EXTRA="-r P"
```

Can we just add it inline here?

```suggestion
      # Run the tests
      - name: Test with pytest (Linux/macOS)
        run: |
          echo ${GITHUB_WORKSPACE}/gmt-install-dir/bin >> $GITHUB_PATH  # Add GMT's bin to PATH
          make test PYTEST_EXTRA="-r P"
```

---

I don't think it will work.

> Prepends a directory to the system PATH variable and makes it available to all subsequent actions in the current job; the currently running action cannot access the updated path variable.

https://docs.github.com/en/actions/reference/workflow-commands-for-github-actions#adding-a-system-path

---

Ah, that's annoying, nevermind then.